client/src/modules/cohort/components/CohortHeader.jsx

```diff
@@ -23,10 +23,10 @@ class CohortHeader extends PureComponent {
     const trimmedDescription = description.trim();
 
     this.state = {
-      description: trimmedDescription,
+      descriptionInput: trimmedDescription,
```

Comment:
it's more like desciprionValue and nameValue.
Cause descriptionInput more to a DOM element rather than value of an input.